test/e2e-test-application/cypress/e2e/tests/1-angular/luigi-client-link-manager-features-6.cy.js

```diff
@@ -0,0 +1,102 @@
+import { wrap } from "module";
+
+
+describe('Luigi Client linkManager Modal', () => {
+  beforeEach(() => {
+    cy.visitLoggedIn('/');
+  });
+
+    it('print stuff out so, I can check what functions are available etc.', () => {
+      cy.window().then(win => {
+        cy.log(win)
+        cy.log("----win.Luigi----")
+        cy.log(win.Luigi)
+        cy.log("----win.Luigi.navigation()----")
+        cy.log(win.Luigi.navigation())
+        cy.log("----win.Luigi.navigation().openAsModal() without cy.wrap----")
+        cy.log(win.Luigi.navigation().openAsModal('/projects/pr2'))
+
+        const p = win.Luigi.navigation().openAsModal('/projects/pr2')
+        cy.wrap(p).then((res, rej) => {   // Question: if .then() is only executed after the promise is fulfilled, why is this part executed even though the modal has not been closed yet?
```

Comment:
I think this would not work because
`Luigi.navigation().openAsModal(...).then`  does not exist.

I think the [openAsModal](https://docs.luigi-project.io/docs/luigi-core-api/?section=examples-28) on Luigi Core rather takes an extra parameter as a onCloseCallback function instead of returning a promise